docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -158,10 +158,6 @@ The new DaemonSet with the different template will recognize all the existing Po
 matching labels.  It will not modify or delete them despite a mismatch in the Pod template.
 You will need to force new Pod creation by deleting the Pod or deleting the node.
 
-In Kubernetes version 1.6 and later, you can [perform a rolling update](/docs/tasks/manage-daemon/update-daemon-set/) on a DaemonSet.
```

Comment:
Should we keep this line?

Reply:
Seems reasonable, yes. I've just restored it.